src/scss/components/_menu.scss

```diff
@@ -27,7 +27,7 @@
   &--over {
     width: 100%;
     position: fixed;
-    z-index: 2;
+    z-index: 10;
```

Comment:
@bsoares yes... I think we need to create a map and get function to organize all this z-index